Add initial version of 3scale alert SOPs included on 3scale v2.9 (#9)
3scale/3scale-Operations: sops/alerts/replication_controller_replicas_mismatch.adoc

:toc:
:toc-placement!:

= ThreescaleReplicationControllerReplicasMismatch

toc::[]

== Description

* This alert will trigger if a `ReplicationController` does not have the desired number of pods running

== Troubleshooting

> * Check why new pods might be failing

Reviewer: ```suggestion
* Check if the new pods created by the replicaset might be failing
```

> * Check if maybe there is an issue with the kubernetes scheduler (lack of nodes resources...)

Reviewer: ```suggestion
* Check if there might be an issue with the scheduling due to lack of nodes resources or tolerations
```
